backend/controllers/prisonerProfile/prisonerCellHistory.js

```diff
@@ -82,18 +75,7 @@ module.exports = ({ oauthApi, prisonApi, logError, page = 0 }) => async (req, re
             }
           })
         : [],
-      currentLocation: {
-        establishment: agencyData.find(agencyDetails => currentLocation.agencyId === agencyDetails.agencyId)
-          .description,
-        location: extractLocation(currentLocation.description, currentLocation.agencyId),
-        movedIn: currentLocation.assignmentDateTime && formatTimestampToDateTime(currentLocation.assignmentDateTime),
-        assignmentDateTime: moment(currentLocation.assignmentDateTime).format('YYYY-MM-DDTHH:mm:ss'),
-        assignmentEndDateTime: currentLocation.assignmentEndDateTime
-          ? moment(currentLocation.assignmentEndDate).format('YYYY-MM-DDTHH:mm:ss')
-          : today,
-        livingUnitId: currentLocation.livingUnitId,
-        agencyId: currentLocation.agencyId,
-      },
+      currentLocation,
```

Comment:
`currentLocation` now gets it's structure from `cellData` which removed a bit of duplication.